pom.xml

```diff
@@ -85,6 +85,7 @@
     <properties>
         <awsjavasdk.version>${project.version}</awsjavasdk.version>
         <jackson.version>2.10.4</jackson.version>
+        <jackson.databind.version>2.10.5.1</jackson.databind.version>
```

Comment:
Just noticed that jackson-core doesn't have 2.10.5.1. Can we upgrade jackson-core to `2.10.5` to "match" with databind?

Reply:
Updated..
Also f integ tests, stability tests passed.
Benchmark test is in progress since last 30 mins